nucliadb_protos/python/nucliadb_protos/audit_pb2.pyi

```diff
@@ -3,28 +3,21 @@
 isort:skip_file
 """
 import builtins
-import collections.abc
 import google.protobuf.descriptor
 import google.protobuf.internal.containers
 import google.protobuf.internal.enum_type_wrapper
 import google.protobuf.message
 import google.protobuf.timestamp_pb2
 import nucliadb_protos.nodereader_pb2
 import nucliadb_protos.resources_pb2
-import sys
 import typing
-
-if sys.version_info >= (3, 10):
-    import typing as typing_extensions
-else:
-    import typing_extensions
+import typing_extensions
```

Comment:
We keep having issues with protos versions. Why are all changed again?

Reply:
I am at protobuff 4.22, seems like it's the most up to date